blockstore/src/main/java/brooklyn/location/blockstore/BlockDeviceOptions.java

```diff
@@ -12,7 +12,7 @@
     private String name;
     private String zone;
     private Map<String, String> tags = Maps.newHashMap();
-    private int sizeInGb;
+    private Number sizeInGb;
```

Comment:
When the object is created form a map in some cases sizeInGb can be wrongly coerced to non Integer value and even if the value is of type 1.0 for example, instantiating will fail.
It is also changed for better error reporting when sizeInGb is set to double value like 2.4 for example.